packages/react/src/primitives/CheckboxField/CheckboxField.tsx

```diff
@@ -0,0 +1,78 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import { Checkbox } from '../Checkbox';
+import { FieldErrorMessage, FieldDescription } from '../Field';
+import { Flex } from '../Flex';
+import { Label } from '../Label';
+import { CheckBoxFieldProps } from '../types/checkboxField';
+import { ComponentClassNames } from '../shared';
+
+export const CheckboxField: React.FC<CheckBoxFieldProps> = ({
```

Comment:
I was thinking if this level is necessary. Looks like `Checkbox` is good enough. Just follow the same patterns as how we build other fields.